src/components/file-viewer/file-viewer.tsx

```
@@ -246,6 +257,26 @@ export class FileViewer {
+            <limel-email-viewer
+                subject={this.email?.subject}
+                from={this.email?.from}
+                to={this.email?.to}
+                cc={this.email?.cc}
+                date={this.email?.date}
+                bodyHtml={this.email?.bodyHtml}
+                bodyText={this.email?.bodyText}
+                attachments={this.email?.attachments}
+                fallbackUrl={this.sanitizeUrl(this.fileUrl)}
+                language={this.language}
+            >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)